Logic/Osm/ChangesetHandler.ts

```diff
@@ -352,7 +352,7 @@ export class ChangesetHandler {
             const metadata = [
                 ["created_by", `MapComplete ${Constants.vNumber}`],
                 ["locale", Locale.language.data],
-                ["host", window.location.host],
+                ["host", `${window.location.origin}${window.location.pathname}`],
                 ["path", path],
```

Comment:
If the path is included, line 356 (and additionally 350 and 351) can be dropped too